File: src/main/java/com/willmadison/legacycodekatas/fulfillment/warehouse/management/WarehouseManagement.java
package com.willmadison.legacycodekatas.fulfillment.warehouse.management;

public interface WarehouseManagement {

    OrderVerificationSearchResponse search(OrderVerificationSearchRequest request);

    PickSearchResponse search(PickSearchRequest reqeust);

    PickSaveResponse save(PickSaveRequest request);
}

File: src/main/kotlin/com/willmadison/legacycodekatas/fulfillment/warehouse/management/WarehouseManagement.kt
package com.willmadison.legacycodekatas.fulfillment.warehouse.management

interface WarehouseManagement {
    fun search(searchRequest: OrderVerificationSearchRequest): OrderVerificationSearchResponse
    fun search(searchRequest: PickSearchRequest): PickSearchResponse
    fun save(request: PickSaveRequest): PickSaveResponse
}